src/Parsehub/Parsehub.php

```diff
@@ -401,4 +416,40 @@ public function getProjectRunDeleteApiUrl($run_token)
         $url = $api_url . '/runs/' . $run_token . '?api_key=' . $api_key;
         return $url;
     }
+
+    /**
+    * Get http status code of url
+    * @param string $url url to check
+    * @return integer http status
+    */
+    public function getHttpStatusCode($url)
+    {
+        $handler = curl_init($url);
```

Comment:
Rather then using curl you can use `head` method of phphttpclient library which we are using to get the header and check the status code. see http://phphttpclient.com/docs/source-class-Httpful.Request.html#1065 see complete doc here http://phphttpclient.com/docs/
One extra point: you can use `followRedirect` method of Httpful\Request class if url redirect you to some other url. I used this method when downloading html of a page.